packages/model/src/discover.ts

```diff
@@ -0,0 +1,8 @@
+import { PageResponse } from './page';
+import { UserResponse } from './user';
+
+export interface DiscoverResponse {
+  pages: ReadonlyArray<PageResponse>;
```

Comment:
Why do we call this pages for the FE but guidance for the BE?

Reply:
I created guidance first on the CMS and on the frontend the component is expecting pages. I'm changing the CMS property name.